components/ChallengeMaterial.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import ChallengeTitleCard from '../components/ChallengeTitleCard'
+import ChallengeQuestionCard from '../components/ChallengeQuestionCard'
+
+const ChallengeMaterial: React.FC = () => {
+  return (
+    <div className="row">
+      <div className="col-4">
+        <ChallengeTitleCard challengeNum={1} title="Less Than or Equal to 5" />
```

Comment:
I think `components` folder is getting cluttered with too many files. Wanna put `ChallengeTitleCart` and `ChallengeQuestionCard` in this file since they are not used anywhere else?